src/rawchannel.cc

```diff
@@ -303,6 +320,7 @@ class RawChannel : public Nan::ObjectWrap
    */
   static NAN_METHOD(Send)
   {
+    int requiered_mtu = 1;
```

Comment:
Not used in this function, right?

Reply:
correct, first version merge CAN & CAN_FD but it's not a good idea. It's why I create a send_FD() function
Code is removed now